starter-core/src/test/groovy/io/micronaut/starter/feature/messaging/mqtt/MqttSpec.groovy

```diff
@@ -60,7 +60,7 @@ class MqttSpec extends ApplicationContextSpec implements CommandOutputFixture {
         template.contains('''<artifactId>micronaut-maven-plugin</artifactId>
           <configuration>
             <shared>true</shared>
-          </configuration>''') == (dependency == 'mqttv5')
```

Comment:
I had to remove this because config with AOT enabled looks like: 

```xml
    <artifactId>micronaut-maven-plugin</artifactId>
          <configuration>
            <shared>true</shared>
            <configFile>aot-${packaging}.properties</configFile>
          </configuration>
```